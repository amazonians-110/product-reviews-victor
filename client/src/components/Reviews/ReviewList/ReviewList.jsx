import React from 'react';

class ReviewList extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return(
      <div>ReviewList component</div>
    )
  }
}

export default ReviewList;