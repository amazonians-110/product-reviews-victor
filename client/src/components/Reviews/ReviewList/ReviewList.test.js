import React from 'react';
import { shallow } from 'enzyme';

import ReviewList from './ReviewList';

describe('ReviewList', () => {
  it('should render correctly', () => {
    
    const component = shallow(<ReviewList/>);

    expect(component).toMatchSnapshot();
  });
});
