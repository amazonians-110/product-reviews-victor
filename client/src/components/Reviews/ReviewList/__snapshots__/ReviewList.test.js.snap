// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewList should render correctly 1`] = `
<div>
  <div>
    <h6>
      Showing 1-8 of 453 reviews
    </h6>
    <select
      name="reviews"
    >
      <option
        value="Top Reviews"
      >
        Top Reviews
      </option>
      <option
        value="Most recent"
      >
        Most recent
      </option>
    </select>
  </div>
  <div>
    Review of product
  </div>
  <div>
    Review of product
  </div>
  <div>
    Review of product
  </div>
  <div>
    Review of product
  </div>
  <div>
    Review of product
  </div>
</div>
`;
